Crash: kotlin.UninitializedPropertyAccessException: lateinit property cameraBasicHandler has not been initialized

Found a crash when testing #36 with Don't Keep Activities enabled.

```
java.lang.RuntimeException: Unable to start activity ComponentInfo{com.automattic.portkey/com.automattic.portkey.compose.ComposeLoopFrameActivity}: java.lang.RuntimeException: Failed to call observer method
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2951)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3086)
...
     Caused by: java.lang.RuntimeException: Failed to call observer method
        at androidx.lifecycle.ClassesInfoCache$MethodReference.invokeCallback(ClassesInfoCache.java:226)
        ...
     Caused by: kotlin.UninitializedPropertyAccessException: lateinit property cameraBasicHandler has not been initialized
        at com.automattic.photoeditor.state.BackgroundSurfaceManager.getStateFromBundle(BackgroundSurfaceManager.kt:242)
        at com.automattic.photoeditor.state.BackgroundSurfaceManager.onCreate(BackgroundSurfaceManager.kt:55)
        at java.lang.reflect.Method.invoke(Native Method)
        at androidx.lifecycle.ClassesInfoCache$MethodReference.invokeCallback(ClassesInfoCache.java:219)
        at androidx.lifecycle.ClassesInfoCache$CallbackInfo.invokeMethodsForEvent(ClassesInfoCache.java:194) 
...
2019-08-22 08:48:04.855 1152-2939/? W/ActivityManager:   Force finishing activity com.automattic.portkey/.compose.ComposeLoopFrameActivity
```

Steps to reproduce:
1. Enable 'Don't keep activities' in Developer options
2. Launch the app, and open the compose activity
3. Background the app and return

--- a/photoeditor/src/main/java/com/automattic/photoeditor/state/BackgroundSurfaceManager.kt
+++ b/photoeditor/src/main/java/com/automattic/photoeditor/state/BackgroundSurfaceManager.kt
@@ -52,7 +52,6 @@ class BackgroundSurfaceManager(
     fun onCreate(source: LifecycleOwner) {
         // clear surfaceTexture listeners
         photoEditorView.listeners.clear()
-        getStateFromBundle()
 
         // add fragments
         if (useCameraX) {
@@ -61,6 +60,9 @@ class BackgroundSurfaceManager(
             addHandlerFragmentOrFindByTag(CAMERA2)
         }
         addHandlerFragmentOrFindByTag(VIDEOPLAYER)
+
+        // important: only retrieve state after having restored fragments with addHandlerFragmentOrFindByTag as above
+        getStateFromBundle()
 
         if (isCameraVisible || isVideoPlayerVisible) { photoEditorView.toggleTextureView() }
     }

